build-runtime.sh.in

```diff
@@ -1,19 +1,31 @@
 #! /bin/bash
 
+########################
+# build runtime script #
+########################
+#
+# template to be configured by the CMake build system
+
 if which clang >/dev/null; then
     CC=clang
 else
     CC=gcc
 fi
 
-set -x
+STRIP=strip
```

Comment:
Are there any cases where this is not `strip`?

Reply:
I doubt it.

Reply:
Then let's not introduce a variable for it. Makes the code easier to read.